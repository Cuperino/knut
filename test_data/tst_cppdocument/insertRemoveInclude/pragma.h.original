// Some comments

#pragma once

class Foo {

};
