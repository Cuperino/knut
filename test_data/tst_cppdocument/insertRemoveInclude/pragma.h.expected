// Some comments

#pragma once

#include <bar.h>

class Foo {

};
